test/e2e/app-dir/ecmascript-features/app/page.tsx

```diff
@@ -0,0 +1,67 @@
+'use client'
+import { abc } from './export-as'
```

Comment:
```suggestion
import { "abc" as abc } from './export-as'
```

Reply:
Thanks! Added it in a separate file as it's not supported in TypeScript 👍 https://github.com/vercel/next.js/pull/65613/files#diff-ac6597b52518bb2740cb6d042cbf6eb7f806c1d21fc7558c73d0b4ed313b5910